components/Input/input.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Input</title>
+
+    <link rel="stylesheet" href="../../styles/main.css" />
+  </head>
+  <body>
+    <div class="d-flex flex-col">
+      <label class="input-label-req text-sm" for="email">Email</label>
+      <input
+        type="text"
+        class="text-input w-100 p-2 text-sm"
+        placeholder="abc@gmail.com"
+        required
+      />
+    </div>
+
+    <div class="d-flex flex-col">
+      <label class="text-sm" for="name">Name</label>
+      <input
+        type="text"
+        class="text-input w-100 p-2 text-sm"
+        placeholder="Enter your name"
+        required
+      />
+    </div>
+
+    <div class="d-flex flex-col">
+      <label class="input-label-req text-sm" for="password">Password</label>
+      <input
+        type="text"
+        class="text-input w-100 p-2 text-sm"
+        placeholder="Enter your password"
+        required
+      />
+    </div>
+
+    <div class="input-grp-err d-flex flex-col">
+      <label class="input-label-req text-sm" for="confirm-password"
+        >Confirm Password</label
+      >
+      <input
+        type="text"
+        class="text-input w-100 p-2 text-sm"
+        placeholder="Re-enter password"
+        required
+      />
```

Comment:
You can use a more specific class name for w-100 p-2 text-sm.

Reply:
those are my utility classes. I will create proper documentation after some time so that everyone can understand them